### ISSUE TITLE
pexpect.spawn should use list of arguments

### ISSUE THREAD

**Issue by zachborboa** [kennethreitz/pipenv#81]
`pexpect.spawn` should use list of arguments instead of `str.format`.

```python
import pexpect

path = '/tmp/dir-without-spaces'
child = pexpect.spawn('ls {0}'.format(path))
print(child.read())
child.close()

path = '/tmp/dir with spaces'
child = pexpect.spawn('ls {0}'.format(path))
print(child.read())
child.close()

path = '/tmp/dir with spaces'
child = pexpect.spawn('ls', [path])
print(child.read())
child.close()
```

```
In [1]: run test.py
file1.txt  file2.txt

ls: cannot access '/tmp/dir': No such file or directory
ls: cannot access 'with': No such file or directory
ls: cannot access 'spaces': No such file or directory

file1.txt  file2.txt
```

https://github.com/kennethreitz/pipenv/blob/master/pipenv/cli.py#L626
https://github.com/kennethreitz/pipenv/blob/master/pipenv/cli.py#L667

**Comment by kennethreitz** [comment]
pull requests accepted!

**Comment by nateprewitt** [comment]
Resolved with (8d31698).
